README.md

```diff
@@ -1,161 +1,38 @@
 # IBF-system
 
-## Introduction
+IBF-system is a web app to visualize hazard forecasts. It has a [NestJS backend](./services/API-service) and an [Angular frontend](./interfaces/IBF-dashboard). Read our [wiki](https://github.com/rodekruis/IBF-system/wiki) for [functional](https://github.com/rodekruis/IBF-system/wiki/Features) and [technical](https://github.com/rodekruis/IBF-system/wiki/Software-architecture) information.
 
-**NOTE**: Some background on IBF-terminology (e.g. triggers) is
-expected. This can be expanded on later.
+![IBF portal showing floods in Uganda](https://github.com/user-attachments/assets/6cb909e8-be23-40af-ae7e-e9ccba41db40)
 
-This is the repository for the IBF-system. It includes a.o.:
-
-1. An [API-service (backend)](./services/API-service/)
-
-- which accepts input from various IBF-pipelines, that upload impact forecast data to the IBF-system on regular intervals. (See 'Dependencies' below.)
-- and which lets the IBF-dashboard - or other authorized accounts - retrieve data from the IBF-database.
-
-2. An [IBF-dashboard (frontend)](./interfaces/IBF-dashboard/)
-
-- showing all impact forecast data - either leading to a trigger or not - in the IBF-portal
-
-## Dependencies
-
-- The IBF-dashboard will not show meaningful information (or even load correctly) without impact forecast data being uploaded to it.
-- This data is provided by separate IBF-pipelines, that are not part of this repository, but are strongly connected.
-- See the 510 IBF Project Document for more info and links to the 510-instances of these pipelines per disaster-type.
-- For development/testing purposes, there are mock-endpoints and mock-data available to replace the need for these pipelines. (See 'Load local database with data' below.)
+IBF will not show meaningful information without [forecast data](https://github.com/rodekruis/IBF-system/wiki/Create-a-pipeline). This data is uploaded to the IBF-system through the [IBF API](http://localhost:3000/docs). For demo purposes, the [API-service](./services/API-service) includes [mock data](http://localhost:3000/docs#/---%20mock%2Fseed%20data%20---/ScriptsController_mock) to simulate hazard forecasts.
```

Comment:
is the localhost:3000 linking here a good idea? A consumer might not have set anything up at this point and/or have set up a different port?